pkg/apis/core/types_shoot.go

```diff
@@ -777,7 +777,8 @@ type Worker struct {
 	Annotations map[string]string
 	// CABundle is a certificate bundle which will be installed onto every machine of this worker pool.
 	CABundle *string
-	// CRI contains configurations of CRI support of every machine in the worker pool
+	// CRI contains configurations of CRI support of every machine in the worker pool.
+	// Defaults to a CRI with name `containerd` when the Kubernetes version of the `Shoot` is >= 1.22.
```

Comment:
Nit: `make generate` is required to have this change reflected to `hack/api-reference/core.md` as well.

Reply:
done.